platformcomponents/desktop/attachment-file-default.json

```diff
@@ -0,0 +1,21 @@
+{
+  "attachment-file-default": {
+    "sent": {
+      "comment": "Applied to sent & safe file attachments",
+      "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=5428%3A29",
+      "border": "@theme-outline-secondary-normal",
+      "iconForeground": "@theme-common-text-white",
+      "iconBackground": {
+        "default": "@theme-background-accent-normal"
+      },
+      "textFileName": "@theme-text-primary-normal",
```

Comment:
```
"label":{
     "primary": "@theme-text-primary-normal",
     "secondary": "@theme-text-secondary-normal"
}
```

Reply:
Pushed commit
`Grouped text under label key in attachment-file-default`
Changing this conversation to Resolved.